### PR TITLE
[admin] Priority ordering for templates.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@akatsoulas r?

File: snippets/base/models.py [hunk 1]
```diff
@@ -209,6 +209,9 @@ class SnippetTemplate(CachingMixin, models.Model):
     snippet will fill in.
     """
     name = models.CharField(max_length=255, unique=True)
+    priority = models.BooleanField(
+        'Priority template', default=False,
```
Comment (akatsoulas) [inline snippets/base/models.py]: Please use **kwargs everywhere. eg `verbose_name=Priority template'.

**Comment by akatsoulas** [comment]
r+wc

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**